scripts/microbe_stage/organelle_table.as

```diff
@@ -277,8 +277,7 @@ void setupOrganelles(){
         Int2(-1, 1),
         Int2(-1, 0),
         Int2(1, 1),
-        Int2(0, 2),
-        Int2(-1, 2)
+        Int2(0, 2)
```

Comment:
Why was the size of the nucleus changed? It's been the same size in hexes forever.

Reply:
like now starting cell was 10 hexes with nucleus, if you add organelle nucleus of 10 hexes size it's 11 hexes, i just remove one to make the situation as was before

Reply:
But you changed the configuration of the nucleus? That affects everything starting from the generated collision of the cell and how the membrane wraps around things.

Reply:
My point is that there should be a lot of discussion going into *why* the hexes of the nucleus need to be changed. And until then I'm not going to accept a change here.